internal/featuresupport/features_misc.go

```diff
@@ -266,7 +266,7 @@ func (f *skipMcoReboot) GetName() string {
 }
 
 func (f *skipMcoReboot) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
-	if !isFeatureCompatibleWithArchitecture(f, filters.OpenshiftVersion, swag.StringValue(filters.CPUArchitecture)) {
```

Comment:
isn't that a more generic function? can't we do it inside? as part of the supported features?

Reply:
Maybe